packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -26,9 +27,12 @@ const AccountMenuItems = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const menuItems = MENU_ITEMS.map(item => {
+        const id = `account_menu.${item.name
+            .toLowerCase()
+            .replace(/ /g, '_')}_link`;
```

Comment:
This works, but I think I'd rather just add an `id` property to each object in `MENU_ITEMS` rather than generating it here. Also, I think we should probably move `MENU_ITEMS` to the talon, and return it (under the property `menuItems`). Would that work?

Looks much better with `<FormattedMessage id={id} />` though. 👍